ospfd/ospf_abr.c

```diff
@@ -1706,10 +1706,10 @@ ospf_abr_manage_discard_routes (struct ospf *ospf)
 	  {
 	    if (range->specifics)
 	      ospf_add_discard_route (ospf->new_table, area,
-				      (struct prefix_ipv4 *) &rn->p);
+				      (struct prefix_ipv4 *) &rn->p, ospf);
```

Comment:
For every single function that you have added ospf to the parameter list, please make it the first.